doc/architecture/software-update.rst

```diff
@@ -35,9 +66,7 @@ and consumed by the software update client running on supported devices.
 
 Update information is secured by generating hashes of each file and then
```

Comment:
The missing piece is the handling of the signature in the client. The server is signing both individual manifests and the Manifest of Manifests. I've just built an image with an extra bundle and the corresponding update stream and it has all the files required, plus signatures.